packages/core/src/button/button.stories.ts

```diff
@@ -1,146 +1,130 @@
 import type { ButtonFill, ButtonSize, ButtonVariant } from './button.js';
 import type { Meta, StoryObj } from '@storybook/web-components';
 import '../button-bar/register.js';
+import '../icon/register.js';
 import './register.js';
 import { html } from 'lit';
 
 interface Props {
   fill: ButtonFill;
   size: ButtonSize;
   text: string;
+  icon: string;
   variant: ButtonVariant;
 }
 
 type Story = StoryObj<Props>;
+const fills = ['default', 'outline', 'link'];
+const variants = ['default', 'primary', 'success', 'warning', 'danger'];
```

Comment:
I would at least add typing to these variables, so there no errors in the list: `const variants: ButtonVariant[] = ...`